lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -210,7 +210,7 @@ static int get_uli_from_session_req(
     return 0;
   }
 
-  uli[0] = saved_req->uli.present;
+  uli[0] = 130;  // TAI and ECGI - defined in 29.061
```

Comment:
instead of hardcoding it may be better to check the flags in uli.present field and then pass the values. It would be good to codify the values rather than passing numeric value directly. It is up to you.

Reply:
Good point @ulaskozat , created https://github.com/magma/magma/issues/3051 to track this.